packages/modula-examples/src/index.html

```diff
@@ -1,11 +1,54 @@
 <html>
 	<head>
-		<link rel="stylesheet" href="./spectre.css">
+    <title>Modulajs</title>
+    <link rel="icon" href="./favicon.ico" />
+		<link rel="stylesheet" href="./spectre.css" />
     <style>
+      /* general */
+      a:focus {
+        box-shadow: none;
+      }
```

Comment:
we're now using a mixture of CSS + styled components, do you think we can unify them together?

Reply:
I think it is totally fine to remove existing styled component for the demo page and use pure css if that is easier.